docs/API_POLICY.md

```diff
@@ -0,0 +1,39 @@
+# Hub API Policy
+This document proposes a policy regarding making API updates in this repo.
+
+There are 2 types of APIs:
+## 1. Versioned APIs: 
+- These are the APIs which are exposed for external users to use and integrate with their applications.
+- These will have support provided by the Hub team.
+- These are the APIs which return hub resource data.
+- These are versioned, the version of the APIs will be prefixed to the API route. 
+Eg. resource API - /v1/resources
+- Hub Release Version has nothing to do with API version. API version will be incremented only when there is a breaking change. Making a hub major release not necessarily increment API version too.
+If hub is having a major release to v2.0.0 then it doesn't mean API will be incremented to `/v2`. 
+- Swagger Documentation for these APIs can be found at `https://api.hub.tekton.dev/<version>/schema/swagger.json`. Replace `<version>` with current version of APIs.
+ 
+
+#### Additive changes
+- Additive changes are changes that add to the API and do not cause problems for exsiting users of the API.
+- If the current version of API is `/v1` then any new addition for eg. adding new field in response of a API will not affect existing users of v1 version of that API.
```

Comment:
So in this case there will be an API release and not bumping the version of API right to /v2 ?

Reply:
yes bumping the version to /v2 only if the API is breaking